### PR TITLE
Modify Events

### LINKED ISSUE #18: Modify Events
- For events already registered, the application must allow modification of the event data.
 - An option to modify each event must be provided.

### PR DESCRIPTION
**Pull request by angel23v**
- For events already registered, the application must allow modification of the event data.
- An option to modify each event must be provided.

For more info see: https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/issues/18

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,14 +3,23 @@
...
+  get '/events/index', to: 'events#index'
 
-  get 'events/main', to: 'events#main'
+  get '/events/main', to: 'events#main'
 
   get '/events/new', to: 'events#new'
 
+  # get '/events/show', to: 'events#show'
+
+  get '/events/show/:id', to: 'events#show'
```
Comment (juperameza) [inline config/routes.rb]: Why is show if youre editing????

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -21,9 +20,22 @@ def index
     @event = Event.all
   end
 
+  def show
```
Comment (juperameza) [inline app/controllers/events_controller.rb]: Same here

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

File: app/views/events/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    <div class="text-center">
+        <%= form.submit "Guardar Cambios", class: "btn btn-primary", data: { disable_with: "Editando Evento..." } %>
+    </div>
+  <% end %>
+
+  <div class="modal-footer mt-3">
+      <%= link_to "Regresar", events_path, class: "btn btn-secondary" %>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/events/edit.html.erb]: indent at 2 spaces in this file, please.

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**